mw-webapp/src/logic/userPage/UserPage.tsx

```diff
@@ -46,23 +73,55 @@ export const UserPage = (props: UserPageProps) => {
   return (
     userPreview &&
     <div className={styles.container}>
-      <Title
-        level={HeadingLevel.h2}
-        text={`Page of ${userPreview.name}`}
-      />
+      <div className={styles.row}>
+        <Title
+          text="Name:"
```

Comment:
Please add a space after the colon. Also in the email and description lines.

Reply:
Added